nuxeo-pipeline-apb/templates/nuxeo-image-dockerfile.j2

```diff
@@ -1,7 +1,9 @@
 FROM {{ base_image_name }}:latest
 # Set up the /build folder with the install script, the marketplace package and the studio credentials
 USER root
-COPY marketplace.zip /build/marketplace
+{% if not global_config.studio_only %}
+    COPY marketplace.zip /build/marketplace
```

Comment:
I would remove the indentation, as it will be shown in the resulting `Dockerfile`.